### PR TITLE
SNC - 18 April

### PR DESCRIPTION
**Pull request by paulsnoops**

### CODE REVIEW

**Review by allentiak — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ ManaCost:3 W W
 Types:Creature Human Warrior
 PT:3/5
 T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigSelectTargetCreature | TriggerDescription$ Whenever CARDNAME attacks, CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.
-SVar:TrigSelectTargetCreature:DB$ Pump | ValidTgts$ Creature.YouCtrl+Other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one other target creature | SubAbility$ DBKeywordChoice | SpellDescription$ CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.
+SVar:TrigSelectTargetCreature:DB$ Pump | ValidTgts$ Creature.Other+YouCtrl | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one other target creature | SubAbility$ DBKeywordChoice | SpellDescription$ CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt:6]: I don't think this should really matter, just affects the order Forge sorts things but should arrive at the same results

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt:6]: No need to revert though

**Review by allentiak — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/elegant_entourage.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Elegant Entourage
+ManaCost:3 G
+Types:Creature Elf Druid
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Alliance — Whenever another creature enters the battlefield under your control, target creature other than CARDNAME gets +1/+1 and gains trample until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.Other | TgtPrompt$ Select target creature other than CARDNAME | NumAtt$ +1 | NumDef$ +1 | KW$ Trample
+SVar:PlayMain1:TRUE
+Oracle:Alliance — Whenever another creature enters the battlefield under your control, target creature other than Elegant Entourage gets +1/+1 and gains trample until end of turn.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/elegant_entourage.txt:8]: Also `SVar:BuffedBy:Creature`

File: forge-gui/res/cardsfolder/upcoming/elegant_entourage.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Elegant Entourage
+ManaCost:3 G
+Types:Creature Elf Druid
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Alliance — Whenever another creature enters the battlefield under your control, target creature other than CARDNAME gets +1/+1 and gains trample until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.Other | TgtPrompt$ Select target creature other than CARDNAME | NumAtt$ +1 | NumDef$ +1 | KW$ Trample
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/elegant_entourage.txt:6]: Not sure CARDNAME will get parsed in TgtPrompt... I'd go with `Select another target creature` ?

File: forge-gui/res/cardsfolder/upcoming/girder_goons.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Girder Goons
+ManaCost:4 B
+Types:Creature Ogre Warrior
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ When CARDNAME dies, create a tapped 2/2 black Rogue creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ b_2_2_rogue | TokenTapped$ True
+K:Blitz:3 B
+DeckHas:Ability$Token|Sacrifice
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/girder_goons.txt:8]: DeckHas:Ability$Token|Sacrifice & Type$Rogue

File: forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt (hunk 1 above)
Comment (paulsnoops) [reply forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt:6]: OCD 😅 I just couldn't leave it alone

File: forge-gui/res/cardsfolder/upcoming/pugnacious_pugilist.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Pugnacious Pugilist
+ManaCost:3 R R
+Types:Creature Ogre Warrior
+PT:4/4
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME attacks, create a tapped and attacking 1/1 red Devil creature token with "When this creature dies, it deals 1 damage to any target."
+SVar:TrigToken:DB$ Token | TokenScript$ r_1_1_devil_burn | TokenTapped$ True | TokenAttacking$ True
+K:Blitz:3 R
+DeckHas:Ability$Token|Sacrifice
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/pugnacious_pugilist.txt:8]: & Type$Devil

File: forge-gui/res/cardsfolder/upcoming/social_climber.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Social Climber
+ManaCost:2 G
+Types:Creature Human Druid
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigGainLife | TriggerDescription$ Alliance — Whenever another creature enters the battlefield under your control, you gain 1 life.
+SVar:TrigGainLife:DB$ GainLife | Defined$ You | LifeAmount$ 1
+DeckHas:Ability$LifeGain
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/social_climber.txt:7]: also SVar:BuffedBy:Creature

File: forge-gui/res/cardsfolder/upcoming/witty_roastmaster.txt [hunk 7]
```diff
@@ -0,0 +1,7 @@
+Name:Witty Roastmaster
+ManaCost:2 R
+Types:Creature Devil Citizen
+PT:3/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigDmg | TriggerDescription$ Alliance — Whenever another creature enters the battlefield under your control, CARDNAME deals 1 damage to each opponent.
+SVar:TrigDmg:DB$ DealDamage | Defined$ Player.Opponent | NumDmg$ 1
+Oracle:Alliance — Whenever another creature enters the battlefield under your control, Witty Roastmaster deals 1 damage to each opponent.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/witty_roastmaster.txt:7]: SVar:BuffedBy:Creature

**Review by Northmoc — APPROVED** [review]

**Review by Northmoc — APPROVED** [review]
🥳

**Pull request merged**